commerce/bookstore/python/load_data.py

```diff
@@ -0,0 +1,280 @@
+                                           "insert $th (sub-tag: $b, sup-tag: $p) isa tag-hierarchy;")
+                transaction.query().insert("insert $g 'History' isa genre-tag;")
+                transaction.query().insert("match $b = 'History'; $b isa genre-tag;"
+                                           "$p = 'Non fiction'; $p isa genre-tag;"
+                                           "insert $th (sub-tag: $b, sup-tag: $p) isa tag-hierarchy;")
+                transaction.query().insert("insert $g 'Politics' isa genre-tag;")
+                transaction.query().insert("match $b = 'Politics'; $b isa genre-tag;"
+                                           "$p = 'Non fiction'; $p isa genre-tag;"
+                                           "insert $th (sub-tag: $b, sup-tag: $p) isa tag-hierarchy;")
+                transaction.query().insert("insert $g 'Up to 5 years' isa genre-tag;")
+                transaction.query().insert("match $b = 'Up to 5 years'; $b isa genre-tag;"
+                                           "$p = 'Kids friendly'; $p isa genre-tag;"
+                                           "insert $th (sub-tag: $b, sup-tag: $p) isa tag-hierarchy;")
+                transaction.query().insert("insert $g 'Technical Documentation' isa genre-tag;")
+                transaction.query().insert("match $b = 'Technical Documentation'; $b isa genre-tag;"
+                                           "$p = 'Non fiction'; $p isa genre-tag;"
+                                           "insert $th (sub-tag: $b, sup-tag: $p) isa tag-hierarchy;")
+                transaction.query().insert("match $b = 'Technical Documentation'; $b isa genre-tag;"
+                                           "$p = 'Adults only'; $p isa genre-tag;"
+                                           "insert $th (sub-tag: $b, sup-tag: $p) isa tag-hierarchy;")
+                transaction.query().insert("insert $g 'Map' isa genre-tag;")
+                transaction.query().insert("match $b = 'Map'; $b isa genre-tag;"
+                                           "$p = 'Technical Documentation'; $p isa genre-tag;"
+                                           "insert $th (sub-tag: $b, sup-tag: $p) isa tag-hierarchy;")
+                transaction.commit()
+    print("Created genre tags.")
+    return
+
+
+def load_data():  # Main data load function
+    create_genre_tags()  # Creating genre tags before loading data
+    with TypeDB.core_client("localhost:1729") as client:
+        with client.session(db, SessionType.DATA) as session:
+            for input in Inputs:  # Iterating through all CSV files
+                input["file"] = data_path + input["file"]
+                if debug: print("Loading from [" + input["file"] + ".csv] into TypeDB ...")
+                load_data_into_typedb(input, session)  # Main data loading function. Repeat for only file in Inputs
+            # generate_orders()  # Add randomly generated lists of items into orders
```

Comment:
In general we shouldn't include commented code in a complete example. Here's a couple of possibilities.

1. simple TODO
```suggestion
            # TODO: Add randomly generated lists of items into orders
```

2. TODO that references an issue we raise on GitHub in this repo
```suggestion
            # generate_orders()  # TODO: fix this method (#165)
```

Reply:
That is not a TODO. That's a rudimentary leftover. To be deleted.

Reply:
Fixed that in https://github.com/vaticle/typedb-examples/pull/143/commits/bd33d9ea0e7a8e096c563a965b848d8a07c6468d

Reply:
Thank you for suggesting using issue mentions in the ToDo's. I have not yet tried this.